tsconfig.json

```diff
@@ -27,7 +23,6 @@
     "strict": true,
     "target": "es5"
   },
-  "include": [
```

Comment:
The old "include" configuration seems a lot simpler than the new. What motivated this change?

Reply:
I changed it back to src, this was there just to prevent errors caused by non-overlap in tslint and eslint